pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -133,6 +136,23 @@ func EnforceHostNetwork() bool {
 	return cephv1.EnforceHostNetwork()
 }
 
+func SetrevisionHistoryLimit(data map[string]string) {
+	strval := k8sutil.GetValue(data, revisionHistoryLimitSettingName, "")
+
+	numval, err := strconv.Atoi(strval)
+	if err != nil {
+		logger.Warningf("failed to parse value %q for %q. assuming default value 3.", strval, revisionHistoryLimitSettingName)
+		return
+
+	}
```

Comment:
```suggestion
if strval != ""{
numval, err := strconv.Atoi(strval)
	if err != nil {
		logger.Warningf("failed to parse value %q for %q. assuming default value 3.", strval, revisionHistoryLimitSettingName)
		return

	}
	revisionHistoryLimit = int32(numval)
}
```

can we have something like this to avoid getting this warning if user didn't have this setting in the configmap?